### ISSUE TITLE
Make a more diverse dataset for the example excel file

### ISSUE THREAD

**Issue by kaylai** [kaylai/VESIcal#16]
The example_data.xlsx file should have a more diverse array of sample types! Right now it's a bunch of rhyolites...

**Comment by kaylai** [comment]
Possibly make multiple datasets. For example: a melt inclusion set, a basalt set, a rhyolite set.

**Comment by gordon5m** [comment]
I think making multiple data sets is more useful and enlightening. Plotting results for a bunch of compositions would only be interesting if someone wanted to know how the solubility changed as a function of composition, which is a one-off sort of thing. I have some of Roggensacks's basaltic-ish MI data sets from my RiMG chapter already in XL, as well as some Bishop Tuff MI's (high silica rhyolite) from Wallace's group. Would be nice to have something intermediate as well but, unless you know of something, there aren't any "good" MI data I'm aware of. Maybe something experimental, like my fave Mas-12? Could also use Carroll's phonolite data as well, to get some alkalis in there?

**Comment by kaylai** [comment]
Gordon, this is a great idea. Can you pull together a few excel spreadsheets? I think having MI and also some experimental sets is great, to cover the compositional range!

**Comment by gordon5m** [comment]
Hmm. Github is not letting me attach an XL file here. Should I put it some place else?

**Comment by gordon5m** [comment]
[roggensack_EPSL_CN.xlsx](https://github.com/kaylai/MagmaSatPlus/files/4645655/roggensack_EPSL_CN.xlsx)

**Comment by gordon5m** [comment]
NVM...weird.

**Comment by kaylai** [comment]
I've created a folder called "testDataSets" where I will start dumping these. We can rename them to something more sensible later. Anyone please send any other datasets you might have!

**Comment by PennyWieser** [comment]
I started compiling some of these for basaltic melt inclusions this weekend specifically for datasets that have used published models- I took some data from Tucker et al. (2019). who use Dixon (1997) to get PSat for different hawaiian melt inclusions to then put into the CO2 equation of state method, and some from Emma Bennet's nature paper, who use magmasat. The thermometer Emma said she uses though doesn't match yet so I'm just waiting for her to respond to that email, then will update it so the magmasat outputs match hers exactly.
